LabExT/Experiments/StandardExperiment.py

```diff
@@ -115,6 +114,29 @@ def read_parameters_to_variables(self):
         self.param_output_path = str(self.save_parameters["Raw output path"].value)
         makedirs(self.param_output_path, exist_ok=True)
 
+    def ask_user_to_continue_even_if_live_viewer_active(self):
+        """ check if any of the instrument addresses in the ToDo queue are currently active in live viewer """
+        instr_addrs_in_todo_queue = set()
+        for todo in self.to_do_list:
+            for v in todo.measurement.selected_instruments.values():
+                instr_addrs_in_todo_queue.add(v['visa'])
+        instr_active_in_lv = set()
+        if self._experiment_manager.live_viewer_model is not None:
+            for _, card in self._experiment_manager.live_viewer_model.cards:
+                if card.instrument is not None and card.card_active.get():
+                    visa = card.instrument.instrument_parameters['visa']
+                    if visa in instr_addrs_in_todo_queue:
+                        instr_active_in_lv.add(visa)
+        if instr_active_in_lv:
+            if 'no' == messagebox.askquestion('Active LiveViewer Instruments Found!', 'Some ToDos will interact ' +  
+                                          'with instruments that are currently active in the live viewer. Concurrent instrument ' + 
+                                          'access might not be supported for these instrument classes. It is recommended to ' +
+                                          'stop all live-viewer cards before proceeding. Are you sure you want to proceed?',
+                                          default='no',
+                                          icon='warning'):
+                return False
+        return True
+
```

Comment:
Feature added in this PR: when clicking "Run", there is a message popping up if you forgot to pause the live viewer.

The implementation checks if there is a live viewer card active and occupying an instrument that will be used during the measurements. Only then, the warning is shown.